var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -90,7 +90,12 @@ class Octopus(Package, CudaPackage):
     # TODO: etsf-io, sparskit,
     # feast, libfm, pfft, isf, pnfft, poke
 
-    def install(self, spec, prefix):
+    def autoreconf(self, spec, prefix):
+
+        autoreconf("--install", "--verbose", "--force")
```

Comment:
Is this really necessary? It seems the releases have a `configure` file, why regenerate it?

Reply:
@fangohr That's the default behavior of autoreconf:

 https://github.com/spack/spack/blob/9a51d42cecfdedd9e5776824a420efdf9877d95e/lib/spack/spack/build_systems/autotools.py#L539-L544

Reply:
As @alalazo has already mentioned, it should be possible to simply remove the `autoreconf` method. Moreover, adding `when="@develop"` to the autoconf, automake, libtool and m4 dependencies avoids them for release versions.

Reply:
That's perfect - thank you for the suggestion. We'll get onto it (@iamashwin99) and report back.